src/main/java/com/kelinci/mainapp/api/SimpleUserDatabase.java

```diff
@@ -0,0 +1,41 @@
+package com.kelinci.mainapp.api;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+public class SimpleUserDatabase {
+
+    private static final  List<OurUser> userDatabase = new ArrayList<>();
+
+    public static List<OurUser> getUserDatabase() {
```

Comment:
raczej nie chcemy dawać dostępu do naszej bazy danych. Właśnie po to stworzyliśmy tą klasę SimpleUserDatabase, by ona zwystawiała potrzebne klientom metodki

Reply:
hm, tutaj bym potrzebował calla na 5 min - bo to chyba znaczy, że musimy zrobić konkretny refaktor w całej aplikacji. Czyli getUserDatabase() zwraca jakby za dużo?